tests/director/tests/runtime_contexts_api_test.go

```diff
@@ -260,8 +260,15 @@ func TestRuntimeContextSubscriptionFlows(stdT *testing.T) {
 
 		t.Log("Assert provider runtime is visible in the consumer's account after successful subscription")
 		consumerAccountRuntimes := fixtures.ListRuntimes(t, ctx, certSecuredGraphQLClient, subscriptionConsumerAccountID)
```

Comment:
Instead of List you could get the runtime and verify it exists?
```suggestion
		consumerAccountRuntime := fixtures.GetRuntime(t, ctx, certSecuredGraphQLClient, subscriptionConsumerAccountID, providerRuntime.ID)
```

Reply:
Good catch